benchmarks/dynamo/microbenchmarks/operatorbench.py

```diff
@@ -132,7 +131,7 @@ def skip_operator(operator):
         print(f"Skipping {operator}, no inductor impl")
         return True
 
-    if inductor_config.triton.convolution == "aten" and "convolution" in str(operator):
+    if "convolution" in str(operator):
```

Comment:
Hi @jansel , the changes of this opbenchmark scripts also need for [release 2.0](https://github.com/pytorch/pytorch/blob/release/2.0/benchmarks/dynamo/microbenchmarks/operatorbench.py#LL135), do we need to add this change into release 2.0 branch also?

Reply:
Yes, you're right, it will not be included in binaries.